backend/webgpu/operators/unary.ts

```diff
@@ -38,30 +39,30 @@ export const sigmoid = unary<"f32">("return 1.0 / (1.0 + exp(-1.0 * a));");
 export const square = unary<"f32">("return a * a;");
 export const cos = unary<"f32">("return cos(a);");
 export const cosh = unary<"f32">(`
-  let e2x = exp(-a);
-  return (e2x + 1.0 / e2x) / 2.0;
+let e2x = exp(-a);
+return (e2x + 1.0 / e2x) / 2.0;
 `);
 export const sin = unary<"f32">("return sin(a);");
 export const sinh = unary<"f32">(`
-  let e2x = exp(a);
-  return (e2x - 1.0 / e2x) / 2.0;
+let e2x = exp(a);
+return (e2x - 1.0 / e2x) / 2.0;
 `);
 export const tan = unary<"f32">("return tan(a);");
 export const tanh = unary<"f32">(`
-  let e2x = exp(-2.0 * abs(a));
-  return sign(a) * (1.0 - e2x) / (1.0 + e2x);
+let e2x = exp(-2.0 * abs(a));
+return sign(a) * (1.0 - e2x) / (1.0 + e2x);
 `);
 export const exp = unary<"f32">("return exp(a);");
 export const elu = unary<"f32">(`
-  if (a >= 0.0) {
-    return a;
-  } 
-  return (exp(a) - 1.0);
+if (a >= 0.0) {
+  return a;
+} 
+return (exp(a) - 1.0);
 `);
-export const log = unary<"f32">(`
-  if (a < 0.0) {
-    return 1.0 / 0.0;
-  }
-  return log(a);
+export const log = unary<"f32">((type) => `
+if (a < 0.0) { 
+  return 1.0/0${type}; 
```

Comment:
No need for it to be using the `type` as it is f32 only, change it back to `0.0`